PanKanapka/.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.7" project-jdk-type="Python SDK" />
```

Comment:
Może dodać misc.xml do .gitignore? (Btw, wszyscy mamy pythona 3.6.4, z tego co pamiętam; nie wiem, czy to sprowokuje jakiekolwiek problemy, ale może lepiej trzymać się jednej wersji?).

Reply:
dodam to .gitignore